Bottom: 89552ef3de09c3f2fe804817ef92c90cc92594bf
Top:    b1483242052e0a265e1ae367e928b937a33f9e27
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2010-06-05 14:01:30 +0100

Refresh of add-support-for-a-poured-objec-I

---

diff --git a/src/hid/gcode/gcode.c b/src/hid/gcode/gcode.c
index c3c2e4e..86bac04 100644
--- a/src/hid/gcode/gcode.c
+++ b/src/hid/gcode/gcode.c
@@ -281,7 +281,7 @@ gcode_choose_groups ()
     {
       layer = &PCB->Data->Layer[n];
 
-      if (layer->LineN || layer->TextN || layer->ArcN || layer->PolygonN)
+      if (layer->LineN || layer->TextN || layer->ArcN || layer->PourN)
 	{
 	  /* layer isn't empty */
